CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Changelog
 ----------
 
+v0.2.2
+----
+ - Added Reinforcement Learning example (examples/ppo).
+ - Fix Adafactor bug that prevented factorization.
+ - Fix scan broadcast issue in functional core.
+ - Fix initialization RNGs to work with omnistaging for jitted inits.
+ - Replaces usage of 'param' kind to 'params' collection.
+ - Fix LARS optimizer for zero param initialization.
+ - Added various examples in Linen API (in directory linen_examples).
```

Comment:
Can you add a link to flax/linen/README for others to learn more?

Reply:
Done.